src/webextension/background/messagePorts.js

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { fakeStore as datastore } from "./datastore";
+import datastore from "./datastore";
```

Comment:
@jimporter In case this is not expected, I pulled your last commit and now get an error when trying to invoke the Lockbox browser action:

```
Extension error: TypeError: __WEBPACK_IMPORTED_MODULE_0__datastore__.datastore is undefined moz-extension://10bb0a62-cc55-b740-84fe-776b3feefe02/background.js 37515
[[Exception stack
openLockbox@moz-extension://10bb0a62-cc55-b740-84fe-776b3feefe02/background.js:37515:1
Current stack
applySafeWithoutClone@resource://gre/modules/ExtensionCommon.jsm:287:133
runSafeWithoutClone@resource://gre/modules/ExtensionCommon.jsm:257:12
fire@resource://gre/modules/ExtensionChild.jsm:805:71
withHandlingUserInput@resource://gre/modules/ExtensionUtils.jsm:162:12
receiveMessage@resource://gre/modules/ExtensionChild.jsm:807:18
_callHandlers/<@resource://gre/modules/MessageChannel.jsm:589:17
_callHandlers@resource://gre/modules/MessageChannel.jsm:588:14
_handleMessage/deferred.promise<@resource://gre/modules/MessageChannel.jsm:656:7
_handleMessage@resource://gre/modules/MessageChannel.jsm:653:24
receiveMessage@resource://gre/modules/MessageChannel.jsm:159:5
```